script.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+mysql -u "root"  -e "CREATE DATABASE atlas"
```

Comment:
Not needed. Look at the [mysql image](https://hub.docker.com/r/_/mysql) and again -- tag it at a specific version.